packages/aws-cdk/test/integ/cli/cli.integtest.ts

```diff
@@ -484,6 +484,11 @@ integTest('cdk diff --fail with multiple stack exits with if any of the stacks c
   await expect(fixture.cdk(['diff', '--fail', fixture.fullStackName('test-1'), fixture.fullStackName('test-2')])).rejects.toThrow('exited with error');
 }));
 
+integTest('cdk diff --security-check-only exits when security changes are present', withDefaultFixture(async (fixture) => {
+  const stackName = 'iam-test';
+  await expect(fixture.cdk(['diff', '--security-check-only', fixture.fullStackName(stackName)])).rejects.toThrow('exited with error');
```

Comment:
This test is going to fail. It doesn't set up a difference at all, so what is it going to check we have differences in?

I have to say, I'm a little mystified looking at the other diff tests on what exactly they do, but we should copy the struture from there at least.

Reply:
O snap.. well I tested this on an actual CDK repo and it works HAHA, but yes I'll make this better

Reply:
I took this from the cdk deploy security changes one

Reply:
Wait I also ran this by itself and it worked?